signing/signer.go

```diff
@@ -19,7 +19,6 @@ const (
 	ATX Domain = iota
 	BEACON_FIRST_MSG
 	BEACON_FOLLOWUP_MSG
-	BEACON_PROPOSAL
 	BALLOT
 	HARE
 	POET
```

Comment:
is it possible to make everything except beacon compatible with testnet05? otherwise we can't deploy this change

Reply:
maybe iota should be dropped and we should just manually assign indexes, iota is too easy to break

Reply:
Before https://github.com/spacemeshos/go-spacemesh/pull/4460 the concrete values were:

```golang
const (
	ATX domain = 0
	BEACON = 1
	BALLOT = 2
	HARE = 3
	POET = 4
)
```

I can change it to

```golang
const (
	ATX domain = 0

	BALLOT = 2
	HARE = 3
	POET = 4

	BEACON_FIRST_MSG = 10
	BEACON_FOLLOWUP_MSG = 11
)
```

would this work or does it need to be compatible with the changes introduced in the previous PR?

Reply:
> would this work or does it need to be compatible with the changes introduced in the previous PR?

i think it should be enough. we didn/t deploy previous change